src/app/attendance/components/AttendanceList.tsx

```diff
@@ -0,0 +1,164 @@
+"use client";
+
+import React, { useEffect } from "react";
+
+import {
+  ColumnDef,
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/components/ui/table";
+
+import { useToast } from "@/hooks/use-toast";
+import { Attendance, Member } from "@/types";
+import { UserStatusBadge } from "@/common/StatusBadge/UserStatusBadge";
+import { StandardPage } from "@/common/StandardPage/StandardPage";
+import { Plus } from "lucide-react";
+import MemberTableActionRender from "@/app/members/components/MemberActionRender";
+
+interface AttendanceProps {
+  attendance: {
+    data: Attendance[];
+    success: boolean;
+    message: string;
+  };
+  members: {
+    data: Member[];
+    success: boolean;
+    message: string;
+  };
+}
+
+export const AttendanceList = ({ attendance, members }: AttendanceProps) => {
+  const { toast } = useToast();
+  const { data: attendanceData, success } = attendance;
```

Comment:
Why do we need `success` here?

Reply:
Fixed. I initially used an older approach, but now I've updated it to use the latest method.